src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -775,8 +775,7 @@ public void EndBuild()
 
                     if (_threadException != null)
                     {
-                        // Unfortunately this will reset the callstack
-                        throw _threadException;
```

Comment:
nit: did you search the codebase for `throw` followed by something other than `new`?

Reply:
I hadn't, but this was a great idea. I found a couple more suspicious locations but nothing that obviously fit into this pattern. I'm going to call it Good Enough for Now.

(I did search for `Unfortunately this will reset the callstack`; this was all of those.)